src/inbox.rs

```diff
@@ -41,6 +41,11 @@ impl<A> Inbox<A> where A: Actor
 		(self.id, self.handle.clone())
 	}
 
+	pub fn addr( &self ) -> Addr<A>
+	{
+		Addr::new(self.sender())
+	}
+
```

Comment:
This can't stay, because `Inbox` works for any type that implements `thespis::Address`, not just `Addr`. The idea is that people can swap specific parts of thespis_impl whilst remaining compatible with other thespis code.

Have a look at the [no_rt](https://github.com/thespis-rs/thespis_impl/blob/dev/examples/no_rt.rs) example of how you can have full control. I cleaned it up a bit in the dev branch.